### PR TITLE
Create Environments Promotions modal

### PR DESCRIPTION
**Pull request by ysheng-mdsol**

### CODE REVIEW

File: src/components/TopPanel.js [hunk 1]
```diff
@@ -1,10 +1,108 @@
...
 
-const TopPanel = props => (
-  <div className="top-panel">
-    <div className="top-panel__element">Data Intake Library</div>
-    <div className="top-panel__element"> Environments </div>
-  </div>
-);
+Modal.setAppElement('#root');
+
+const dataPackages = [
```
Comment (bran-van-d) [inline src/components/TopPanel.js]: Nice mock data!

File: src/components/TopPanel.js [hunk 2]
```diff
@@ -1,10 +1,108 @@
...
+
+  closeModal = () => {
+    this.setState({ modalIsOpen: false });
+  };
+
+  render() {
+    return (
+      <div className="top-panel">
+        <button className="top-panel__element">Data Intake Library</button>
+        <button onClick={this.openModal} className="top-panel__element">
```
Comment (rmorenodelpino-mdsol) [inline src/components/TopPanel.js]: I think the convention for naming methods that are executed from events is 'handleWhatever', so handleOpenModal.

File: src/App.css [hunk 3]
```diff
@@ -49,15 +46,53 @@ tfoot {
...
-tr:nth-child(even) {
+#userTable tr:nth-child(even) {
   background-color: rgb(215, 215, 215);
 }
 
 .top-panel__element {
   padding: 15px;
   width: 300px;
   font-size: 18px;
+  background-color: transparent;
```
Comment (rmorenodelpino-mdsol) [inline src/App.css]: Not now but we need to fix the css of the project or this will get out of hand quickly.

File: src/components/UserTable.js [hunk 4]
```diff
@@ -42,7 +42,7 @@ class UserTable extends Component {
         <div>
           <TopPanel />
           <div className="data-table">
-            <table>
+            <table id="userTable">
```
Comment (rmorenodelpino-mdsol) [inline src/components/UserTable.js]: We try to avoid using css based on html elements ids.

File: src/components/TopPanel.js (hunk 2 above)
Comment (ysheng-mdsol) [reply src/components/TopPanel.js]: Will rename.

File: src/App.css (hunk 3 above)
Comment (ysheng-mdsol) [reply src/App.css]: Absolutely!

Comment (bvandyck-mdsol) [reply src/App.css]: Yea i was curious if we should focus too much on that for this POC. Once we have the real application we're going to be using John's styles so this likely wont be an issue

File: src/components/UserTable.js (hunk 4 above)
Comment (ysheng-mdsol) [reply src/components/UserTable.js]: What about using class, like `data-table` in this case?

File: src/components/TopPanel.js (hunk 2 above)
Comment (bvandyck-mdsol) [reply src/components/TopPanel.js]: Yea that sounds right -- but is that a React convention overall, or just something they do on their site? `handle` all over the place might get annoying after a while

File: src/components/TopPanel.js (hunk 1 above)
Comment (ysheng-mdsol) [reply src/components/TopPanel.js]: Thanks!

File: src/components/UserTable.js (hunk 4 above)
Comment (rmorenodelpino-mdsol) [reply src/components/UserTable.js]: Classes is what we normally do. Check the ABEM naming convention for css.

**Pull request closed without merging**